docs/windows-design.md

```diff
@@ -0,0 +1,249 @@
+# Running Antrea on Windows
+
+Antrea supports running on Windows worker Node. In this case, the Windows Node should
+be a worker in the cluster, while the master Node must be Linux. On Windows Node, 
+Antrea sets up an overlay network to forward packets between Nodes and implements
+NetworkPolicies. 
+
+## Design
+
+On Windows, the Host Networking Service (HNS) is a necessary component to support container
+networking. For Antrea on Windows, “Transparent” mode is chosen for the HNS network. In this
+mode, containers will be directly connected to the physical network through an **external** Hyper-V switch. 
+
+OVS is working as the extension of the external Hyper-V switch, which is bound with the HNS 
+Network. Hence, the packets that are sent from/to the containers could be processed by OVS.
+The network adapter used in the HNS Network is also added into the OVS bridge as the uplink
+interface. An interface for the OVS bridge is created, and the original networking
+configuration(e.g., IP, MAC and route entries) on the host network adapter are moved to
+this new interface. Some extra OpenFlow entries are needed to ensure the host traffic can be
+forwarded correctly.
+
+<img src="/docs/assets/hns_integration.svg" width="600" alt="HNS Integration">
+
+SNAT based on OpenFlow is needed to make sure the containers could access the external address.
+The SNATed address is using the IP configured on the OVS bridge. Some assisted OpenFlow entries are
```

Comment:
not sure what you meant to say with "assisted" here

maybe: "Some additional OpenFlow entries are installed to assist in identifying and forwarding the external traffic."